spec/specfile

```diff
@@ -1,5 +1,6 @@
 type WeatherResponse {
     BaseMessage  BaseMessage
+    WeatherConfigWeatherConfig
```

Comment:
Is it a problem that there's no space between the name and type?

Reply:
tf how did that happen

Reply:
This is the specfile so its kinda actually wrong... hmm... Not gonna fix it rn but thats something to think about (specgen does not flatten structs)